<launch>
  <node name="data_visualization_node" pkg="height_mapping" type="data_visualization_node" output="screen">
    <param name="data_directory" value="$(find height_mapping)/maps/" />
    <param name="publish_rate" value="100.0" />
    <param name="frame_id" value="map" />
  </node>

  <!-- Launch RViz with configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find height_mapping)/rviz/data_visualization.rviz" />
</launch> 